testsuite/tests/apicast/test_limit_exceeded_metric.py

```diff
@@ -30,17 +30,24 @@ def application(application):
     return application
 
 
+@pytest.fixture(scope="module")
+def exceed_limit(api_client):
+    """Exceed limit of /anything/exceeded endpoint"""
```

Comment:
This looks pretty weird, thus requires good description why it should be done this way. As it is part of this PR it looks obvious that this is to have this test working after persistence load, however once in the codebase, it won't be evident and there is no relation to persistence (yet I may be totally wrong and the reason is completely different)